tasks/notify.py

```diff
@@ -101,7 +106,15 @@ def send_message(_, notification_type="merge", print_to_stdout=False):
         if print_to_stdout:
             print(f"Would send to {channel}:\n{str(message)}")
         else:
-            send_slack_message(channel, str(message))  # TODO: use channel variable
+            release_branch_regex = re.compile(r"^[0-9]+\.[0-9]+\.[0-9]+.*$")
```

Comment:
I don't think this works as expected. To give a few examples, this regex:
- doesn't match the `7.52.x` release branch
- matches the `7.52.0` release tag
- matches the `7.52.0-rc.1` release tag
- matches the `7.52.0-beta-test-feature` tag

Which means that:
- messages about merge pipelines on the `7.52.x` release branch will be sent to the author (but we want them to be public)
- messages about a deploy pipeline on `7.52.0` or `7.52.0-rc.1` will be sent to the channels (OK)
- messages about a deploy pipeline on the `7.52.0-beta-test-feature` tag will be sent to the channels (but we want them to be sent to the author).